src/hardware/input/mouse_config.cpp

```diff
@@ -160,13 +160,21 @@ static void SetSensitivity(const std::string_view sensitivity_str)
 	const auto& user_default = mouse_predefined.sensitivity_user_default;
 	const auto default_value = coeff * user_default;
 
+	auto set_default_mouse_sensitivity_setting = []() {
+		set_section_property_value("mouse",
+		                           "mouse_sensitivity",
+		                           format_str("%d", user_default));
+	};
+
 	// Split input string into values
 	auto values_str = split(sensitivity_str, " \t,;");
 	if (values_str.size() > 2) {
 		LOG_WARNING("MOUSE: Too many values in 'mouse_sensitivity', using '%d'",
 		            user_default);
 		mouse_config.sensitivity_coeff_x = default_value;
 		mouse_config.sensitivity_coeff_y = default_value;
+
+		set_default_mouse_sensitivity_setting();
```

Comment:
GitHub won't let me comment on the correct line but you're missing a reset to default on line 184 just under this.

Reply:
Thanks, and there's another one here; will address that one too:

```cpp
	if (out_of_range) {
		LOG_WARNING("MOUSE: 'mouse_sensitivity' adjusted to range %+d - %+d",
		            value_min,
		            value_max);
	}
```